src/internal/operators/mergeMap.ts

```diff
@@ -140,7 +140,8 @@ export class MergeMapSubscriber<T, R> extends OuterSubscriber<T, R> {
 
   private _innerSub(ish: ObservableInput<R>, value: T, index: number): void {
     const innerSubscriber = new InnerSubscriber(this, undefined, undefined);
-    this.add(innerSubscriber);
+    const destination = this.destination as any as Subscription;
```

Comment:
destination is a `Subscriber`, which extends `Subscription`... I'm not sure why any casting is necessary here... :thinking:

Reply:
Nope, it's a `PartialObserver`:

```ts
protected destination: PartialObserver<any>;
```

Reply:
Changing `destination` to be:

```ts
protected destination: PartialObserver<any> | Subscriber<any>;
```

means the type assertion becomes:

```ts
const destination = this.destination as Subscription;
```